services/galley/src/Galley/API/LegalHold.hs

```diff
@@ -42,6 +42,10 @@ createSettings (zusr ::: tid ::: req ::: _) = do
     assertLegalHoldEnabled tid
 
     membs <- Data.teamMembers tid
+    let zothers = map (view userId) membs
+    Log.debug $ Log.field "targets" (toByteString (show zothers))
```

Comment:
```suggestion
    Log.debug $ Log.field "targets" (show $ toByteString <$> zothers)
```

Reply:
(you're still using show on the `Id` type, which is fine for debugging, but the `ToByteString` instance is more concise.)

(speaking of which: why debug?  are customers going to want to use this?  then it should at least be `Info`.)

Reply:
Don't do `show . toByteString`, `toByteString` is sufficient.